DFP Integrated Training Updates (Stress Testing/Benchmarks) (#924)
nv-morpheus/Morpheus: examples/digital_fingerprinting/production/morpheus/benchmarks/README.md

...
```

> When using the MRC SegmentModule in a pipeline, they will also require a module configuration which does gets generated within the test. Additional information is included in the [Morpheus Pipeline with Modules](../../../../../docs/source/developer_guide/guides/6_digital_fingerprinting_reference.md#morpheus-pipeline-with-modules)

Reviewer: ```suggestion
When using the MRC SegmentModule in a pipeline, it will also require a module configuration which gets generated within the test. Additional information is included in the [Morpheus Pipeline with Modules](../../../../../docs/source/developer_guide/guides/6_digital_fingerprinting_reference.md#morpheus-pipeline-with-modules)
```
